lib/components/Dropdown.jsx

```diff
@@ -96,6 +108,7 @@ export default class Dropdown extends Component {
           value={this.state.value}
           onChange={this._handleChange.bind(this)} >
             {emptyPlaceholder}
+            {options}
```

Comment:
I like what you did here, but we should probably use either the options data or the children, not both.  It would be unclear what the behavior _should_ be for supporting both.  My first thought is that having children should trump supplying options data.  Thoughts?